client/pages/paperwork/view.js

```diff
@@ -58,11 +58,10 @@ class Paperwork extends Component {
 
   static renderQuote = (quote, index) => (
     <li key={index}>
-      {quote.message}
-      {Boolean(quote.author) && (
-        <span>
-          - <em>${quote.author}</em>
-        </span>
+      <span className="time">{moment(quote.updatedAt).format('DD MMM, YYYY [\n]HH:mm')}</span>
+      <span className="message">{quote.message}</span>
+      {Boolean(quote.lastAuthor) && (
```

Comment:
We should probably still render the original author's name, but if lastAuthor is set perhaps say something like.. `last edited by: {quote.lastAuthor}`. last

Reply:
Wouldn't it also make sense to keep the original creation date to then? As far as I know the original author is not mentioned outside the API currently.

Reply:
given that _all_ case information should be displayed on this page, it's important that we denote that the quote has been edited. Perhaps we should use the `createdAt` time for the main time, and then maybe have some sort of way of showing if it's been edited, with the details appearing in a box on hover.

Reply:
Will require a bit of a redesign, but as a whole, I agree.